packages/react-components/react-charts-preview/library/src/components/LineChart/useLineChartStyles.styles.ts

```diff
@@ -1,58 +0,0 @@
-import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
```

Comment:
dont remove this file

Reply:
but no styles are used from here. It will be empty then

Reply:
added as discussed